packages/menu/src/MenuItem.ts

```diff
@@ -330,6 +330,9 @@ export class MenuItem extends LikeAnchor(Focusable) {
             flatten: true,
         });
         this.hasSubmenu = this.open || !!assignedElements.length;
```

Comment:
Do a null check on assignedElements first otherwise this can throw a null pointer exception